pages/account/how-to/use-2fa.mdx

```diff
@@ -68,7 +68,7 @@ If you no longer have access to the device in which you set up your MFA, you can
 ## How to disable MFA
 
 <Message type="important">
-  You cannot disable MFA if you are a Member of one or more Organizations where MFA is enforced. If you wish to disable MFA, you must first leave these Organizations. If you do not know which of your Organizations enforce MFA, follow the procedure below until step 3. The Organizations will be listed in the **Disable MFA** pop-up.
+  If you disable MFA as a Member of an Organization that has MFA enforced, you will need to re-enable it before the Organization's [grace period](/iam/how-to/comply-with-sec-requirements-member/#grace-period), otherwise your Member account will be locked.
```

Comment:
```suggestion
  If you disable MFA as a Member of an Organization that has MFA enforced, you will need to re-enable it before the Organization's [grace period](/iam/how-to/comply-with-sec-requirements-member/#grace-period) ends, otherwise your Member account will be locked.
```